Orange/widgets/visualize/owdistributions.py

```diff
@@ -371,8 +371,6 @@ def display_distribution(self):
                 self.tooltip_items.append((self.plot, item))
 
     def _on_relative_freq_changed(self):
-        if not self.distributions:
-            return
```

Comment:
I think you should keep these two lines, too.

Connect File -> Distributions to give some data to Distributions. Now remove the data by deleting the connection. Click Show relative frequencies and the widget will crash. These two lines prevent this.